### PR TITLE
[FIX] getVersion()

### PR DESCRIPTION
**Pull request by 0ddlyoko**
# Description
Testing with MockBukkit & ProtocolLib doesn't work because `Server#getVersion()` always returns 0.1.0 and ProtocolLib use this method to retrieve the [version of Minecraft used](https://github.com/dmulloy2/ProtocolLib/blob/97972acee84ec73c57866ed1d7edbda20f273a40/src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java#L138)
This fix fixes the issue

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Version number updated in `settings.gradle`.
- [ ] Unit tests added.
- [X] Code follows existing code format.

# Info on creating a pull request
Before the pull request can be accepted, the version number stored in settings.gradle should be updated.
MockBukkit uses semantic-versioning (https://semver.org/)

 - If the PR fixes a bug in MockBukkit, update the patch version. (if the current version is `0.4.1`, the new version should be `0.4.2`)
 - If the PR adds a new feature to MockBukkit, update minor version. (if the current version is `0.4.1`, the new version should be `0.5.0`)
 - Make sure that unit tests are added which test the relevant changes.
 - Make sure that the changes follow the existing code format.

Note that a PR that fixes an `UnimplementedOperationException` should be considered a new version and not a bugfix.

The version number can be found in `settings.gradle`.

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/ServerMock.java [hunk 1]
```diff
@@ -426,7 +426,7 @@ public String getName()
 	@Override
 	public String getVersion()
 	{
-		return "0.1.0";
+		return BUKKIT_VERSION;
```
Comment (TheBusyBiscuit) [inline src/main/java/be/seeseemelk/mockbukkit/ServerMock.java]: That's a hard pass from me.
This version does not follow Bukkit's convention, see my comment here:

https://github.com/seeseemelk/MockBukkit/pull/161/files/ddeab6d986b9976cf9a2c465a74499bac213a140#diff-763d283381d2bc7fcc200b07f44349baea512d37c0bb809b7c8b8e087d7eb1b9

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Also please be truthful about what you check on the todo list.
I see you have checked that you added Unit Tests but there were no Unit tests to be found.

File: src/main/java/be/seeseemelk/mockbukkit/ServerMock.java (hunk 1 above)
Comment (0ddlyoko) [reply src/main/java/be/seeseemelk/mockbukkit/ServerMock.java]: Like now ?

**Review by TheBusyBiscuit — APPROVED** [review]
If we were worried about micro-optimizations we could just make it a field and return that.
But I am more than fine with this.

The version number could have been bumped too but we are currently blocked by updating to the latest Spigot API anyway.

**Comment by TheBusyBiscuit** [comment]
Also, I cannot merge this due to the build check not completing yet.
GH Actions servers are currently experiencing some issues, so it may resolve itself by tomorrow.

**Pull request merged**